src/components/TimeseriesPreview/stories/basic.story.mdx

```diff
@@ -0,0 +1,221 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { TimeseriesPreview, TimeseriesPreviewWithoutTheme } from '../TimeseriesPreview';
+import { 
+  decorators,
+  dropdown,
+  valToDisplay,
+  formatValue,
+  nameFormatter,
+  descriptionFormatter,
+  retrieveTimeseries,
+  onToggleVisibility,
+  retrieveLatestDatapoint,
+  styles
+} from './helper';
+import { ThemeProvider } from 'styled-components';
+
+
+<Meta title="Time Series|TimeseriesPreview" decorators={decorators} />
+
+# Timeseries Preview 
+
+Component displays data related to provided `timeseriesId` with latest datapoint available. Latest datapoint value is updated with interval provided via `updateInterval` prop.
+
+Component has embedded logic for fetching timeseries data and latest datapoint available for it. To make it work properly, component
+has to be wrapped into `ClientSDKProvider` component with defined client SDK instance. Fetching logic
+also can be redefined via `retrieveTimeseries` and `retrieveLatestDatapoint` props to retrieve timeseries data and latest datapoint respectively
+
+### Basic usage:
+
+Imports:
+
+```jsx
```

Comment:
`jsx` -> `typescript`

Reply:
For now `typescript` is not working. Therefore all stories there including `jsx` for now.

Reply:
but storybook v5.3.1 support typescript highlighting that's why I'm asking you 😄

Reply:
ah it's working now :) sure I'll update :+1: